src/VfacTmdb/Results/Name.php

```diff
@@ -0,0 +1,86 @@
+<?php declare(strict_types = 1);
+/**
+ * This file is part of the Tmdb package.
+ *
+ * (c) Vincent Faliès <vincent@vfac.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Vincent Faliès <vincent@vfac.fr>
+ * @copyright Copyright (c) 2017-2020
+ */
+
+
+namespace VfacTmdb\Results;
+
+use VfacTmdb\Abstracts\Results;
+use VfacTmdb\Interfaces\TmdbInterface;
+
+/**
+ * Class to manipulate a name result
+ * @package Tmdb
+ * @author Steve Richter <steve@nerdbra.in>
+ * @copyright Copyright (c) 2017-2020
+ */
+class Name extends Results
```

Comment:
Could you rename this class to `AlternativeName`. It's more clear I think.

Reply:
Will rename